js/sts-hotspot.js

```diff
@@ -0,0 +1,72 @@
+AFRAME.registerComponent('sts-hotspot', {
+  schema: {
+    show: { type: 'selector' },
+    sound: { type: 'selector' },
+    playing: { default: false }
+  },
+
+  init: function () {
+    var self = this;
+    var popupEl = this.data.show;
+
+    popupEl.dataset.hotspot = self.el.id;
+
+    self.el.addEventListener('mouseenter', function () {
+      self.el.setAttribute('sts-hotspot', { playing: true });
+      self.el.dataset.playing = true;
```

Comment:
nit: can you store this in the component instead of a `data-` attribute? or is there a particular advantage to using this?

Reply:
I do this because I need to be able to get it globally.  the component is per audio instance.